tests/config/test_model_config.py

```
@@ -46,23 +45,19 @@ def test_default_initialization(default_config):
-    """Test validation failure with an invalid pre_trained_weights."""
-    with pytest.raises(ValueError):
-        ModelConfig(backbone_type="net")
+        ModelConfig(
+            pre_trained_weights="here",
+            backbone_config=BackboneConfig(unet=UNetConfig()),
+        )
 
 
 def test_update_config(default_config):
     """Test updating configuration attributes."""
     config = OmegaConf.structured(
```

Ruff (0.8.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)